packages/excalidraw/components/App.tsx

```diff
@@ -7958,7 +7959,13 @@ class App extends React.Component<AppProps, AppState> {
               elementsToAppend,
               oldIdToDuplicatedId,
             );
-            fixBindingsAfterDuplication(
+            // fixBindingsAfterDuplication(
+            //   nextSceneElements,
+            //   elementsToAppend,
+            //   oldIdToDuplicatedId,
+            //   "duplicatesServeAsOld",
+            // );
+            fixBindingsAfterDuplication2(
```

Comment:
Why a new function? You essentially made modifications to the old function, but it's extremely difficult to see what you changed. Please modify the original function if you only make changes to that, so we can see what exactly changed.